server/src/main/java/io/unitycatalog/server/persist/utils/HibernateUtils.java

```diff
@@ -31,10 +30,9 @@ private static SessionFactory createSessionFactory() {
         throw new RuntimeException("PropertiesUtil instance is null in createSessionFactory");
       }
 
-      Properties hibernateProperties = new Properties();
-      try (InputStream input = Files.newInputStream(Paths.get("etc/conf/hibernate.properties"))) {
-        hibernateProperties.load(input);
-      }
+      // Load properties with environment variable substitution to configure service in Docker
+      Properties hibernateProperties =
+          ConfigLoader.loadProperties(Paths.get("etc/conf/hibernate.properties"));
```

Comment:
Looks like this utility to read `.properties` files with env var substitution relies on https://github.com/spotbugs/spotbugs, which has a  LGPL-2.1 license. This is [failing the ci[(https://github.com/unitycatalog/unitycatalog/actions/runs/10932397607/job/30349161846).

Is this really prohibited?